packages/sonarwhal/docs/user-guide/rules/index.md

```diff
@@ -18,6 +18,7 @@
 * [`html-checker`](../../../../rule-html-checker/README.md)
 * [`meta-viewport`](../../../../rule-meta-viewport/README.md)
 * [`no-friendly-error-pages`](../../../../rule-no-friendly-error-pages/README.md)
+* [`typescript-config-is-valid`](typescript-config-is-valid.md)
```

Comment:
I'd create a new section `Development` for this kind of rules.

Also kind of related https://github.com/sonarwhal/sonarwhal.com/issues/391